enhancements/update/cvo-techpreview-manifests.md

```diff
@@ -52,28 +53,33 @@ resources like namespaces, roles, rolebindings, serviceacounts, and deployments.
     is enforced using validation on the apiserver.
 4. Allow manifests to be removed or un-reconciled if TechPreviewNoUpgrade is *not* set.  There are a few case (mostly
     around CRDs), where this capability is useful, but it is not as common as needing to create something additional.
-5. Allowing manifests condition on other featuresets in this iteration.
 
 ## Proposal
 
-Add an annotation that can be set on CVO manifests called `release.openshift.io/feature-gate` with one legal/honored value
-of "TechPreviewNoUpgrade".
-If the value is not "TechPreviewNoUpgrade", the manifest is never created.
+Add an annotation that can be set on CVO manifests called `release.openshift.io/feature-set` that can be set to a comma
+delimited list that can contain any value from 
+[FeatureSet](https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L28-L43) and "Default" in place of "".
```

Comment:
nit: line references like `#L28-L43` mixed with a floating ref like `master` are brittle.  Can we use [this][1] instead?

[1]: https://pkg.go.dev/github.com/openshift/api/config/v1#FeatureSet